bin/extract-ORFs.py

```diff
@@ -46,13 +46,35 @@
         help='Only ORFs of a maximum of this length will be written to'
         'stdout.')
 
+    group = parser.add_mutually_exclusive_group()
+
+    group.add_argument(
+        '--kozakInfo', default=False, action='store_true',
+        help='A file with Kozak consensus information will be written'
+        'containing all Kozak consensus sequences. Only applicable if DNA'
+        'reads given.')
+
+    parser.add_argument(
+        '--kozakOnly', default=False, action='store_true',
+        help='Only ORFs that also have a Kozak consensus will be written to'
```

Comment:
In this option and other ones you should put a space at the end of each line (before the final `'`) otherwise the words in the help message will be run together.